source/common/stats/thread_local_store.cc

```diff
@@ -335,6 +335,47 @@ StatType& ThreadLocalStoreImpl::ScopeImpl::safeMakeStat(
   return **central_ref;
 }
 
+// TODO(ahedberg): This is largely duplicated from safeMakeStat and should
+// be cleaned up at some point.
+template <class StatType>
+const StatType* ThreadLocalStoreImpl::ScopeImpl::safeGetStat(
+    StatName name, StatMap<std::shared_ptr<StatType>>& central_cache_map,
+    StatMap<std::shared_ptr<StatType>>* tls_cache, StatNameHashSet* tls_rejected_stats) const {
+
+  // We do name-rejections on the full name, prior to truncation.
```

Comment:
I believe there is no longer any stat name truncation. Since this is from safeMakeStat, now would probably be a good time to remove the truncation-related comments in there, as well.

Reply:
Removed comments.